### DISCUSSION TITLE
SEB IOS  Switching Application query

### DISCUSSION THREAD

**Discussion by VarunTwri** [SafeExamBrowser/seb-mac#366] [General]
if the SEB can inform the browser or does when a program change happened (some kind of event)
So that we don’t have to hardcode the password in the SEB but can deal with it in the software itself.

**Comment by danschlet** [comment]
I don't understand your question.

Do you have an issue having to set a quit password in SEB? Obviously there must be a password defined by an admin user so an exam proctor/supporter can quit SEB in case a student gets stuck in the locked state (AAC Assessment Mode active). There are various ways how to configure this password (use a sebs:// link to a .seb configuration file to start an exam, use a sebs:// config link, a SEB config QR code, use MDM to prepare managed iPads with SEB client settings containing the password, send students a .seb file to start the exam or configure client settings, use the Automatic Client Configuration feature in the home WiFi or VPN of an institution. No need to hardcode a quit password!

**Comment by VarunTwri** [comment]
Hi @danschlet,
We wanted to know if we get any event when the application is switched in IOS(here if we don't set quit password we are able to switch between different application using home button) .
so, If we have some events, then we can notify the application and we can log off.
 the pupil must login again to start the application.
